wc-admin.php

```diff
@@ -181,3 +181,20 @@ function wc_admin_woocommerce_updated() {
  * Gutenberg has also disabled emojis. More on that here -> https://github.com/WordPress/gutenberg/pull/6151
  */
 remove_action( 'admin_print_scripts', 'print_emoji_detection_script' );
+
+/**
+ * Filter in our ActionScheduler Store class.
+ *
+ * @return string ActionScheduler Store class name.
+ */
+function wc_admin_set_actionscheduler_store_class() {
+	// Include our store class here instead of wc_admin_plugins_loaded()
+	// because ActionScheduler is hooked into `plugins_loaded` at a
+	// much higher priority.
+	require_once WC_ADMIN_ABSPATH . '/includes/class-wc-admin-actionscheduler-wppoststore.php';
+
+	return 'WC_Admin_ActionScheduler_WPPostStore';
```

Comment:
This should check that the passed value is the AS post data store so it doesn't break a site using a custom store.

Reply:
@jeffstieler Outside of my note above, this looks great and works as expected.

Reply:
Added a class check in 1ca9bf1.